src/app/portlets/content-types/edit/content-types-edit.component.html

```diff
@@ -7,48 +7,46 @@
             <div class="content-type__edit-title">
                 <h4>{{ data.name }} </h4>
                 <div class="content-type__edit-subtitle">
-                    <span class="content-type__edit-variable" *ngIf="isEditMode()">
+                    <span class="content-type__edit-variable">
                         {{ dotMessageService.get('contenttypes.content.variable') }}: {{ data.variable }} 
                         <span class="content-type__edit-dot-separator">•</span> {{ dotMessageService.get('contenttypes.form.identifier') }}: {{ data.id }}
                     </span>
                 </div>
             </div>
         </div>
-        <div *ngIf="isEditMode()">
-            <div class="content-type__edit-actions">
-                <button type="button" pButton flat label="Edit" (click)="editForm()" icon="fa-edit" id="form-edit-button"></button>
-            </div>
+        <div class="content-type__edit-actions">
+            <button type="button" pButton flat label="Edit" (click)="show = true" icon="fa-edit" id="form-edit-button"></button>
         </div>
     </div>
-    <p-dialog
-        modal="modal"
-        width="500"
-        [dismissableMask]="true"
-        [(visible)]="show"
-        [responsive]="true"
-        [resizable]="false"
-        [draggable]="false"
-        [closable]="isEditMode()"
-        [height]="dialogHeight"
```

Comment:
Dialogs NEEDS to be responsive, we can't set the height as big as the window if the dialog content doesn't need to.